packages/cozy-client/types/types.d.ts

```diff
@@ -193,6 +193,27 @@ export type CordovaWindow = {
     resolveLocalFileSystemURL: Function;
     handleOpenURL: Function;
 };
+/**
+ * - A document
+ */
+export type CouchDBDocument = {
+    /**
+     * - Id of the document
+     */
+    _id?: string;
+    /**
+     * - Current revision of the document
+     */
+    _rev?: string;
+    /**
+     * - When the document has been deleted
+     */
+    _deleted?: string;
```

Comment:
Is it a string or a bool?

Reply:
Good question. I copy/pasted the `CozyClientDocument` type but it's strange to see a `string` type for this attribute.
I'll check it out.

Reply:
It's definitely a boolean :)